precompiles/bank/method_deposit.go

```diff
@@ -94,37 +94,37 @@ func (c *Contract) deposit(
 	//   this way we map ZRC20 addresses to cosmos denoms "zevm/0x12345".
 	// - Mint coins to the fungible module.
 	// - Send coins from fungible to the caller.
-	coinSet, err := createCoinSet(ZRC20ToCosmosDenom(zrc20Addr), amount)
+	coinSet, err := precompiletypes.CreateCoinSet(precompiletypes.ZRC20ToCosmosDenom(zrc20Addr), amount)
```

Comment:
Couldn't we have `CreateCoinSet` taking directly the zrc20addr and convert it in the function?